raw-data/UW/uw_inventories.tsv

```diff
@@ -0,0 +1,55 @@
+				a	low central vowel	ã	"Oral vowels can undergo nasalization… Nasality in this instance could be argued to be a suprasegmental feature that serves to  mark the plural of certain nouns."	"Vowels additionally are subject to a harmony process in the person prefixes and the negative suffix."
+				u		u	"Oral vowels can undergo nasalization… Nasality in this instance could be argued to be a suprasegmental feature that serves to  mark the plural of certain nouns."	"Vowels additionally are subject to a harmony process in the person prefixes and the negative suffix."
+				u		ũ	"Oral vowels can undergo nasalization… Nasality in this instance could be argued to be a suprasegmental feature that serves to  mark the plural of certain nouns."	"Vowels additionally are subject to a harmony process in the person prefixes and the negative suffix."
+				o		o	"Oral vowels can undergo nasalization… Nasality in this instance could be argued to be a suprasegmental feature that serves to  mark the plural of certain nouns."	"Vowels additionally are subject to a harmony process in the person prefixes and the negative suffix."
+				o		õ	"Oral vowels can undergo nasalization… Nasality in this instance could be argued to be a suprasegmental feature that serves to  mark the plural of certain nouns."	"Vowels additionally are subject to a harmony process in the person prefixes and the negative suffix."
+				<ə>	only appears in past tense suffix /-tə/			
+				ĩ				"Vowels additionally are subject to a harmony process in the person prefixes and the negative suffix."
+				ẽ				"Vowels additionally are subject to a harmony process in the person prefixes and the negative suffix."
+				ɨ̃				"Vowels additionally are subject to a harmony process in the person prefixes and the negative suffix."
+				ã	low central nasal vowel			"Vowels additionally are subject to a harmony process in the person prefixes and the negative suffix."
+				ũ				"Vowels additionally are subject to a harmony process in the person prefixes and the negative suffix."
+				õ				"Vowels additionally are subject to a harmony process in the person prefixes and the negative suffix."
+				tʰ		tʰ	"The nasal consonant [n̥] occurs variably in the data in opposition to a [tʰ] in the 3PL person prefix of verbs when the following vowel is nasal... However, nasalization of /tʰ/ is obligatory in nasal contexts in roots like ĩtʰĩ ‘son’..."	
+				tʰ		n̥	"The nasal consonant [n̥] occurs variably in the data in opposition to a [tʰ] in the 3PL person prefix of verbs when the following vowel is nasal... However, nasalization of /tʰ/ is obligatory in nasal contexts in roots like ĩtʰĩ ‘son’..."	
+				pʰ		pʰ	"all instances of this nasal occur before a nasal vowel... and there are no instances of /pʰ/ before a nasal vowel"	
```

Comment:
put the note about the nasal allophone only on the line where the nasal allophone occurs (and similarly throughout)